BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+mmorpg.com##.div-84937-desktop
+mmorpg.com##.tabs--fill_last
```

Comment:
In my opinion this rule is too generic, it needs to be made more specific.

Reply:
Maybe `.sidebar > div.tabs--fill_last` is ok